eiBoard/src/main/java/dhbw/eiCompany/service/EntryService.java

```diff
@@ -0,0 +1,15 @@
+package dhbw.eiCompany.service;
+
+import dhbw.eiCompany.database.Entry;
+
+import java.util.List;
+
+public interface EntryService {
+
+    void saveOrUpdate(Entry entry);
```

Comment:
saveOrUpdate entfernen bzw splitten in save und in Update

Reply:
resolved